sdk_contrib/aws_support/build.gradle

```diff
@@ -0,0 +1,19 @@
+description = 'OpenTelemetry SDK AWS Support'
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+dependencies {
+    api project(':opentelemetry-api'),
+            project(':opentelemetry-sdk')
+
+    implementation libraries.guava,
+            libraries.disruptor
+
+    implementation 'com.amazonaws:aws-xray-recorder-sdk-core:2.4.0'
+    implementation 'com.amazonaws:aws-java-sdk-core:1.11.701'
```

Comment:
will any of this need to change for the 2.x SDK? If so, we might want to label this as a 1.x version in the module and description.

Reply:
Package names are completely different in the v2 SDK. I did some renaming so the AWS SDK which is being used is emphasized.